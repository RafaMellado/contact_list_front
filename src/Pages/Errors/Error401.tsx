export function Error401() {
  return <h1>Error 401</h1>;
}
