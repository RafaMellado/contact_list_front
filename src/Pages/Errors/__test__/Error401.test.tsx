import { Error401 } from "../Error401";
import { act, render } from "@testing-library/react";

describe(`<${Error401.name} />`, () => {
  const factoryComponent = () => render(<Error401 />);

  test("renders correctly the component", () => {
    const component = factoryComponent();

    act(() => {
      expect(component.asFragment()).toMatchSnapshot();
    });
  });
});
