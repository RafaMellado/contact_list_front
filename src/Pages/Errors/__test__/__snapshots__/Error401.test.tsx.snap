// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error401 /> renders correctly the component 1`] = `
<DocumentFragment>
  <h1>
    Error 401
  </h1>
</DocumentFragment>
`;
